src/components/i18nStrings.vue

```diff
@@ -101,7 +101,8 @@ export default {
 	methods: {
 		addPair(lang) {
 			if (!lang || lang in this.state) return
-			this.$set(this.state, lang, '')
+			this.$set(this.state, lang, '')			
+			this.$store.commit('setLanguages', {[lang]:true})
```

Comment:
what is this 'setLanguages', where is the counter part in the source tree?

Reply:
Mutation method